File: DSA/Priyanshi/Day_1_question_2.java
import java.util.*;

public class Day_1_question_2
{
    public static int countPairs(int[] arr, int k) {
        int count = 0;
        for (int i = 0; i < arr.length; i++) {
            for (int j = i + 1; j < arr.length; j++) {
                if ((arr[i] + arr[j]) % k == 0) {
                    count++;
                }
            }
        }
        return count;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n=0;
        System.out.println("Enter the size of array: ");
        n = sc.nextInt();

        int arr[] = new int[n];

        System.out.println("Enter the elements of array: ");
        for(int i=0;i<n;i++)
        {
            arr[i] = sc.nextInt();
        }

        int k = 3;
        int result = countPairs(arr, k);
        System.out.println("Number of valid pairs: " + result);
    }
}

File: DSA/Pragya_Agarwal/Day1_Question2.py
arr = [1, 3, 2, 6, 1, 2]
k = 3

result = 0

for i in range(len(arr)):
    for j in range(i + 1, len(arr)):
        if (arr[i] + arr[j]) % k == 0:
            result += 1

print("The number of (i,j) pairs:",result) 
